cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -521,11 +521,10 @@ def render_files():
     render('kube-defaults.defaults', '/etc/default/kube-defaults', context)
 
     # when files change on disk, we need to inform systemd of the changes
-    try:
```

Comment:
Yeah, its performing the same function as this block with check_call

by piping it through call we can just run it and if it fails subprocess didnt five a fig. so why do more loc?